service.py

```diff
@@ -24,9 +25,6 @@ def handler(event, context):
         logger.error('Records block contains no records')
         raise NoRecordsReceived('Records block empty', event)
 
-    # Method to be invoked goes here
-    # TODO Implement oauth checking
-
     results = parseRecords(records)
```

Comment:
Since all parseRecords() does is `list(map(parseRecord, records))`, it seems you could skip the extra function call.

Reply:
You're right, but I think that having the one-liner makes reading this slightly easier (for me at least), I'm open to disagreement on that though

Reply:
FWIW another reason to do this is so you can mock `parseRecords` separately during tests.